src/main/java/uk/gov/ons/ssdc/caseprocessor/model/dto/UacUpdateDTO.java

```diff
@@ -7,10 +7,9 @@
 
 @Data
 @JsonInclude(Include.NON_NULL)
-public class UacDTO {
+public class UacUpdateDTO {
```

Comment:
Missing uacHash from here?

Reply:
Those are being handled on a separate ticket, they need some more thought and/or changes to other features.